lib/matplotlib/axes/_axes.py

```diff
@@ -5345,10 +5345,10 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
 
         origin : {'upper', 'lower'}, default: :rc:`image.origin`
             Place the [0, 0] index of the array in the upper left or lower
-            left corner of the axes. The convention (the default) 'upper' is
+            left corner of the Axes. The convention (the default) 'upper' is
             typically used for matrices and images.
 
-            Note that the vertical axes points upward for 'lower'
+            Note that the vertical Axes points upward for 'lower'
```

Comment:
Actually this should have been a singular:
```suggestion
            Note that the vertical axis points upward for 'lower'
```

Reply:
Thank you, I should have caught that!